apps/fly-my-rockets/src/app/shared/shell/shell.component.scss

```diff
@@ -52,5 +52,7 @@ i {
 .main-content {
   max-width: 920px;
   margin: 24px auto;
-  padding: 24px 12px;
+  // 68px bottom padding + 24px bottom margin to make sure nothing
+  // is obscured by the FAB
+  padding: 24px 12px 68px 12px;
```

Comment:
`68px` comes from adding 56px (the height of the FAB) to the standard 12px padding.